coverage.txt

```diff
@@ -1,87 +1,5 @@
 
 mode: atomic
-github.com/jesseduffield/lazydocker/pkg/commands/container.go:247.64,249.2 1 0
```

Comment:
is this file supposed to be committed?

Reply:
No but because i ran the ./test.sh it gave me this what i accidentally commited because of a `git add .`